app/views/people/show.html.haml

```diff
@@ -2,8 +2,12 @@
   .container
     .block--border__bottom
       - if Pugin::Feature::Bandiera.show_list_images?
-        - if @person.image_id(show_placeholder:false)
-          %img{src: "#{ENV['IMAGE_SERVICE_URL']}/#{@person.image_id}.jpg?crop=2&width=732px", alt: "#{@person.display_name}"}
+        - if @person.image_id(show_placeholder:true)
+          %figure
+            %img{src: "#{ENV['IMAGE_SERVICE_URL']}/#{@person.image_id}.jpeg?crop=CU_5:2&width=1464&quality=80", alt: "#{@person.display_name}"}
+            %span
+              = "Image loading"
+              %object.loading{class: "loading", data: "https://s3-eu-west-1.amazonaws.com/web1devci.pugin-website/1.4.4/icons/loading.svg", type: "image/svg+xml", tabindex: "1"}
```

Comment:
Tabindex should be -1;
Need to add aria-hidden='true' also so that screenreaders do not pick this up